src/lib/seam/use-seam-client.ts

```diff
@@ -11,8 +12,10 @@ export function useSeamClient(): {
   error: unknown
 } {
   const { client, clientOptions, publishableKey, ...context } = useSeamContext()
-  const [clientSession, setClientSession] = useClientSession()
-  const userIdentifierKey = useUserIdentifierKey(context.userIdentifierKey)
+  const [clientSession, setClientSession] = useState<ClientSession | null>(null)
```

Comment:
It's possible we may not even need `useState` here anymore since `useQuery` should manage the lifecycle for all our request data.

Reply:
Updated in https://github.com/seamapi/react/pull/99